app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -29,47 +28,52 @@ def manage
 
   # POST /wrangling_guidelines
   def create
-    @wrangling_guideline = WranglingGuideline.new(params[:wrangling_guideline])
+    @wrangling_guideline = WranglingGuideline.new(wrangling_guideline_params)
 
     if @wrangling_guideline.save
       flash[:notice] = ts('Wrangling Guideline was successfully created.')
       redirect_to(@wrangling_guideline)
     else
-      render :action => 'new'
+      render action: 'new'
     end
   end
 
   # PUT /wrangling_guidelines/1
   def update
-    @wrangling_guideline = WranglingGuideline.find(params[:id])
+    @wrangling_guideline = WranglingGuideline.find(wrangling_guideline_id)
 
-    if @wrangling_guideline.update_attributes(params[:wrangling_guideline])
+    if @wrangling_guideline.update_attributes(wrangling_guideline_params)
       flash[:notice] = ts('Wrangling Guideline was successfully updated.')
       redirect_to(@wrangling_guideline)
     else
-      render :action => 'edit'
+      render action: 'edit'
     end
   end
 
   # reorder FAQs
   def update_positions
     if params[:wrangling_guidelines]
-      @wrangling_guidelines = WranglingGuideline.reorder(params[:wrangling_guidelines])       
+      @wrangling_guidelines = WranglingGuideline.reorder(params.permit(:utf8, :commit, wrangling_guidelines: [])[:wrangling_guidelines])
```

Comment:
Sooooo, for what it's worth, we're not doing [mass assignment](http://guides.rubyonrails.org/v3.2/security.html#mass-assignment) here -- mass assignment is when you're using a parameters hash to update the attributes of a single object (or to create a single object with attributes assigned from such a hash). With `reorder`, we're only updating one attribute (`position`) of multiple `WranglingGuideline` objects. (The relevant code is `list_item.update_attribute(:position, index + 1)}` in `lib/sortable_list`.)

So, anyway, I'm pretty sure the reason we're having trouble here is we _shouldn't_ be trying to use strong parameters here at all. I checked our Brakeman report, and indeed this isn't flagged as something that has a mass assignment vulnerability. I've also tried using `update_positions` to change the titles of wrangling guidelines, and I've had no success.